main.cpp

```diff
@@ -81,8 +81,8 @@ int main(int argc, char *argv[]){
     Game::load();
         
     QSurfaceFormat format;
-    //format.setVersion(3, 3);
-    //format.setProfile(QSurfaceFormat::CoreProfile);
+    format.setVersion(3, 3);

```

Comment:
Force OpenGL to be 3.3, core profile

Reply:
Hi.
I can't apply your patch, because most of the TSRE users still have old GPU with no OpenGL 3.3 support. That's why I keep OpenGL 3.0 at the moment. :(
Does OpenGL 3.0 no longer work with MacOS?

Reply:
Also, ET is no longer a TSRE support forum. I was banned there, so don't use this forum anymore.

Reply:
Let me try 3.0; But core profile is required.

Reply:
No problem. You can add  #ifdef __APPLE__ here.
The problem is with shader files. They have to be version 130. If version 130 doesn't work for you, then maybe try to modify your solution so there is a new shaders33 directory in app data directory with OpenGL 3.3 shaders and that it is used instead of shaders directory if apple build is used.